README.md

```diff
@@ -1 +1,184 @@
+- **[Like a movie.]**
+- **[Show the number of likes on a espisode.]**
+- **[Leave user comments on a movie.]**
+- - **[Display the number of Tvshows]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a> -->
+
+## 📹 Video Presentation <a name="video-presentation"></a>
+- Coming Soon
+- [Video Presentation](https://www.loom.com/share/8489447b36fb4be0ae7e317ae1e9e78e?sid=bed2b077-e524-453d-b1cf-44b2b071142f)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To all the new developers this project will give you an idea how to start building your mobile vision apps
+To get a local copy up and running, follow these steps.
+follow me on my github account at richie1988 then clone or fork the repository called mobile-portfoliofinish
+
+### Prerequisites
+
+In order to run this project you need:
+desktop-computer/laptop
+any text editor installed inside
+
+### Setup
+Clone this repository to your desired folder:
+```sh
+cd my-folder
+Clone this repository to your desired folder:
+run:
+git clone (https://github.com/richie1988/KanbanCapstone)
+
+### Install
+Install the prerequisites this project using:
+```sh
+cd Restaurant-System
+```
+```bash
+npm start
+```
+
+### Usage
+To run the project:
+```bash
+npm install
+```
+### Run tests
+To run tests after setting up the linters, run the following command:
+```sh
+npx hint .
+```
+```sh
+npx stylelint "**/*.{css,scss}"
+```
+```bash
+npx eslint .
+```
+### Deployment
+
+You can deploy this project using:
+.Vscode
+.github
+.git
+.any text editor
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="Richard Sikaonga"></a>
+
+I would like Thank Microverse Team and my Collabaration partners whom they have paired me for the entire project the contributed there knowledge and ideas towards the Contact form part for this projects.
+
+👤 **Author**
+ **Richard Sikaonga**
+- GitHub: [@githubhandle](https://github.com/richie1988)
+- Twitter: [@twitterhandle](@Richiesikaonga)
+- LinkedIn: [LinkedIn](in/richard-sikaonga-039940275)
+
+👤 **Author2**
+ **Ismail Halliru Muhammad**
+- GitHub: [@githubhandle](https://github.com/ismailhallirumuhammad)
+- Twitter: [@twitterhandle](https://mobile.twitter.com/IsmailhalliruM1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/mwlite/in/ismail-halliru-muhammad-2a8453127)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[Consumption of more APIs]**
+- **[More dynamic styling]**
+- **[Improved UX interaction]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://docs.github.com/en/issues/tracking-your-work-with-issues/quickstart).
```

Comment:
- [ ] As per the previous reviewer's request, please add the issue link of your current project. This will make it easier for users to contact you. Currently, you are redirecting to the github documentation about how the issues work. Kindly replace it with your current Repo issue link. 🙏